### DISCUSSION TITLE
How to use rewrites to catch _error/404 pages and redirect to a error zone on a multi-zone app?

### DISCUSSION THREAD

**Discussion by joaoalbertocsjunior** [vercel/next.js#42234] [Help]
I've been trying this for the last two or three days, to redirect 404 and _error from other zones to an error zone. Problem is: i don't want to have a 404 and _error page on every zone. 

Supose i have that:

-Main Folder
--Zone 1
--Zone 2
--Error Zone
---404 Page
---_Error Page

Is there a solution for this problem?

**Comment by aggmoulik** [comment]
@joaoalbertocsjunior Have you found the solution, I am looking at same problem.
